da/test/da_test_helpers.go

```diff
@@ -0,0 +1,97 @@
+package test
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+	"os"
+	"strconv"
+	"time"
+
+	tmlog "github.com/tendermint/tendermint/libs/log"
+	"google.golang.org/grpc"
+
+	cmock "github.com/rollkit/rollkit/da/celestia/mock"
+	grpcda "github.com/rollkit/rollkit/da/grpc"
+	"github.com/rollkit/rollkit/da/grpc/mockserv"
+	"github.com/rollkit/rollkit/store"
+	"github.com/rollkit/rollkit/types"
+)
+
+const mockDaBlockTime = 100 * time.Millisecond
+
+func getRandomTx() types.Tx {
+	size := rand.Int()%100 + 100 //nolint:gosec
+	return types.Tx(getRandomBytes(size))
+}
+
+func getRandomBytes(n int) []byte {
+	data := make([]byte, n)
+	_, _ = rand.Read(data) //nolint:gosec
+	return data
+}
+
+// copy-pasted from store/store_test.go
```

Comment:
let's create a follow up issue, if there are copy paste helpers, we should move them to a `test/helpers_test.go` so that the can be reused.

Reply:
https://github.com/rollkit/rollkit/issues/985